src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { getInfoFromTab } from '../../functions/getInfoFromTab';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const isActive = tabs.some(tab => tab.id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map(tab => (
+              <li
+                className={classNames({ 'is-active': isActive === tab.id })}
```

Comment:
when variable name starts from "is" - it means that this variable contains boolean value
so you can name it "activeTabId"

but i can suggest better way

```suggestion
	const selectedTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
	
  return (
    <div data-cy="TabsComponent">
      <div className="tabs is-boxed">
        <ul>
          {
            tabs.map(tab => (
              <li
                className={classNames({ 'is-active': selectedTab.id === tab.id })}
```